docs/src/model_creation/dsl_advanced.md

```diff
@@ -278,10 +278,13 @@ In some cases it may be desirable for Catalyst to not infer species and paramete
 ```@example dsl_advanced_no_infer
 using Catalyst
 # The following case will throw an UndeclaredSymbolicError.
-rn = @reaction_network begin
+try @macroexpand @reaction_network begin
```

Comment:
Why does this need `@macroexpand`? Won't that not run the code but expand it into an expression?

Reply:
So that actually runs the code and displays the correct error? That seems like a strange behavior. Are you sure you shouldn’t be wrapping in an `eval`?

Reply:
Ok, I see. OK I guess the error is not thrown during running of the macro generated expression but during the expression construction, hence why this works.